lib/workers/repository/process/lookup/bucket.ts

```diff
@@ -19,23 +19,60 @@ export function getBucket(
   }
   const fromMajor = versioning.getMajor(currentVersion);
   const toMajor = versioning.getMajor(newVersion);
-  // istanbul ignore if
+
+  // istanbul ignore if: error case
   if (toMajor === null) {
     return null;
   }
+
+  // Check for major update type first
   if (fromMajor !== toMajor) {
     if (separateMultipleMajor) {
-      return `major-${toMajor}`;
+      return `v${toMajor}`;
```

Comment:
this can cause PR closes. so consider this as breaking?

Reply:
I don't think bucket is used today  in branch names